test/e2e/storage/flexvolume_mounted_volume_resize.go

```diff
@@ -43,7 +43,7 @@ import (
 
 const (
 	// total time to wait for cloudprovider or file system resize to finish
-	totalResizeWaitPeriod = 5 * time.Minute
+	totalResizeWaitPeriod = 10 * time.Minute
```

Comment:
If it can take that long, should the tests doing this be marked as "slow"? Or are they typically faster?

Reply:
I am bringing this timeout on par with CSI testsuite - https://github.com/kubernetes/kubernetes/blob/master/test/e2e/storage/testsuites/volume_expand.go#L44

We have observed that, with 5 min timeout - this testsuite flakes a lot whereas CSI testsuite which makes similar API calls to cloudprovider has 100% pass rate - https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-release-master-ci-4.13-e2e-aws-ovn-upgrade/1630466664833224704

Since the CSI testsuite is not marked "Slow", I chose to keep this one same.

Reply:
I was asking with my "SIG Testing Tech lead" hat on :sweat_smile: 

I think the guideline is to mark everything that is close to 5 minutes (average?) as "slow". Several of the storage tests might be candidates for that, it just hasn't been checked.

Anyway, this was just triggered by that change. It can and should be investigated more systematically separately.

Reply:
Agree. We are still investigating this. cc @ConnorJC3 who was helping out with failing API calls on AWS side.